server/journeys/base/controller.ts

```diff
@@ -10,6 +11,9 @@ export class BaseJourneyController {
   createReferral = async (req: Request, createCsipRecordRequest: components['schemas']['CreateCsipRecordRequest']) =>
     this.csipApiService.createReferral(req, createCsipRecordRequest)
 
+  createScreeningOutcome = async (req: Request, payload: Parameters<CsipApiService['createScreeningOutcome']>[1]) =>
+    this.csipApiService.createScreeningOutcome(req, payload)
+
```

Comment:
not sure whether it's good to put it in a shared base class, when this will only be ever used by `screen/check-answers/controller`. 
i think an alternative is to make csipApiService public, so the child controller can use the apiservice to make calls directly.

Reply:
Good point, I agree. Worth making this change for createReferral as well?

Reply:
yea i think so. it's just passing on the function call without performing any logic anyway.